scripts/checkpoint_converters/convert_llama_nemo_to_hf.py

```diff
@@ -143,7 +143,7 @@ def convert(input_nemo_file, output_hf_file, precision=None, cpu_only=False) ->
     ffn_hidden_size = model.cfg.ffn_hidden_size
     num_query_groups = model.cfg.get("num_query_groups", head_num)  # different num_query_groups for 70B
 
-    head_size = hidden_size // head_num
+    head_size = model.cfg.get("kv_channels", hidden_size // head_num)  # equivalent to HF's head_dim
```

Comment:
I think `model.cfg.get("kv_channels", hidden_size // head_num)` will often return `None` because default value in the config is `None`. So safer would be `model.cfg.get("kv_channels") or (hidden_size // head_num)`

Reply:
I modified the line